pac4j-oauth/src/main/java/org/pac4j/scribe/builder/api/WeiboApi20.java

```diff
@@ -0,0 +1,44 @@
+package org.pac4j.scribe.builder.api;
+
+import org.pac4j.scribe.extractors.WeiboJsonExtractor;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.github.scribejava.core.builder.api.OAuth2SignatureType;
+import com.github.scribejava.core.extractors.TokenExtractor;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+
+/**
+ * This class represents the OAuth API implementation for Weibo using OAuth protocol
+ * version 2. It could be part of the Scribe library.
+ * <p>
+ * <p>More info at: <a href="http://open.weibo.com/wiki/Oauth2/authorize">OAuth2.0</a></p>
+ *
+ * @author Zhang Zhenli
+ * @since 3.1.0
+ */
+public class WeiboApi20 extends DefaultApi20 {
```

Comment:
Cannot we use the already available API in Scribe Java: https://github.com/scribejava/scribejava/blob/master/scribejava-apis/src/main/java/com/github/scribejava/apis/SinaWeiboApi20.java instead of creating a new one?

Reply:
Unfortunately, this implementation does not handle uid.
No uid can't get user profile

Reply:
OK. Makes sense